recipes/wasmtime/all/conanfile.py

```diff
@@ -0,0 +1,103 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration, ConanException
+import os
+
+
+class WasmtimeConan(ConanFile):
```

Comment:
You use some "newer" features.
```suggestion
import os

required_conan_version = ">=1.33.0"


class WasmtimeConan(ConanFile):
```

Reply:
Which one ?

Reply:
`tools.get(..., strip_root..., destination=...)`.
That wasn't there in conan 1.32:
https://docs.conan.io/en/1.32/reference/tools.html#tools-get

While updating the recipes on cci, we've also started on requiring this minimum conan version.
It's also a way to push people to upgrade their conan client.

Reply:
I think you still need to add `required_conan_version`.
`strip_root` wasn't in 1.32.

Reply:
Done